src/components.rs

```diff
@@ -75,6 +75,11 @@ pub struct ParticleSystem {
     /// The texture used for each particle.
     pub texture: ParticleTexture,
 
+    /// If provided, re-scale the texture size
+    ///
+    /// Note that this the inherent Sprite size, which can still be scaled up with `Transform`
```

Comment:
I think it would be helpful to note in the comment here that this is just passed directly to  `Sprite::custom_size` as I think that would help clarify the use case.

Reply:
sg!